Update sql-data-warehouse-manage-monitor.md

thomasbragaloni opened the pull request:
removed AND condition or query would return empty results. Also checking memory (using OR instead of AND) wouldn't make sense in a check for transaction log size

azurecla commented:
Hi __@anosov1960__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

thomasbragaloni committed:
Update sql-data-warehouse-manage-monitor.md

removed AND condition or query would return empty results. Also checking memory (using OR instead of AND) wouldn't make sense in a check for transaction log size
